addonscript/requirements/base.py38.txt

```diff
@@ -429,6 +429,7 @@ python-jose==3.3.0 \
     --hash=sha256:9b1376b023f8b298536eedd47ae1089bcdb848f1535ab30555cd92002d78923a
     # via -r requirements/base.in
 pyyaml==6.0 \
```

Comment:
Weird, why do we have new shas for a version of pyyaml that's been out for months?

Reply:
Wow good eye